terraform/digitalgov.gov.tf

```diff
@@ -122,46 +122,35 @@ resource "aws_route53_record" "find_digitalgov_gov_a" {
   ]
 }
 
-# END REDIRECTS
-
-# =================================
-
-# SERVICES
+# U.S. Digital Registry
+# Redirects to cloud.gov External Domains using https://github.com/18f/pages-redirects
+# Contact feedback-analytics@gsa.gov regarding changes
 
 # U.S. Digital Registry
 # usdigitalregistry.digitalgov.gov
-resource "aws_route53_record" "usdigitalregistry_digitalgov_gov_a" {
+resource "aws_route53_record" "redirect_usdigitalregistry_digitalgov_gov" {
```

Comment:
By changing the resource name this is going to create a new record not update the existing resource. It would be cleaner to keep the resource name/id and update the records

Reply:
i hear ya. but leaving this as it makes it non-conistent with the www, and i'd prefer consistency over leaving it just to leave it.  but happy to chat.